transforms/.make.transforms

```diff
@@ -204,3 +201,67 @@ conventions:
 		echo "Recommend not using 'annotator' in your transform name";	\
 	fi
 	@echo Done checking transform project conventions.
+
+.PHONY: .transforms.run-local-ray-sample
+.transforms.run-local-ray-sample:
+	@# Help: Run the $(TRANSFORM_NAME)_local_ray.py file (if it exists).
+	$(MAKE) RUN_FILE=$(TRANSFORM_NAME)_local_ray.py .transforms.run-src-file
+
+.PHONY: .transforms.run-s3-ray-sample
+.transforms.run-s3-ray-sample:
+	@echo ""
+	@echo "Please be sure you've started (make minio-start) and loaded (make minio-load) minio"
+	@echo ""
+	$(MAKE) RUN_FILE=$(TRANSFORM_NAME)_s3_ray.py .transforms.run-src-file
+	@# Help: Run $(TRANSFORM_NAME)_s3_ray.py file (if it exists).  Assumes minio has beens started.
```

Comment:
Although there is help here to start minio and load data, it starts executing regardless and execution can hang for a while retrying S3 calls. I would suggest to remove help and start minio and load data as part of the command

Reply:
I originally had this, but then the minio stuff is almost invisible to the user. and, there is a lot of logging so its hard to see what's really going on.  Thus the change to make the user explicitly manage minio.  Usually i'm a proponent of hiding complexity,  but in this case, I thought it would best to make the user at least aware of minio.

Reply:
You can add echo. I am convinced it should be one

Reply:
If you run it you will see there is one. :)

```
$ make run-s3-ray-sample

Please be sure you've started (make minio-start) and loaded (make minio-load) minio

/Library/Developer/CommandLineTools/usr/bin/make RUN_FILE=noop_s3_ray.py .transforms.run-src-file
source venv/bin/activate;	\
	cd src;				\
	python noop_s3_ray.py 
```

Reply:
And this is causing an issue. People do not read. The start S3 sample and if Minio is not started and there is no data, they complain that it does not work

Reply:
can't do much better than that.  other than doing it all for them.  i'm willing to require this minimum amount of reading in favor of spitting way too much logging at them for all the start, load, run, stop steps.  Also, makes them aware of the steps.

Reply:
That is what I am suggesting. Its easy and will remove all the confusions

Reply:
Added check to be sure minio is running.